### PR TITLE
Fixed arrow display when sorting

### LINKED ISSUE #727: Case Table Sort - clicking a colum header changes the arrow indicator on all columns
**Describe the bug**
Clicking a column header changes the arrow indicator on all columns. Clicking the column header should only change the arrow indicator on the column that is currently the active sorted column.

**To Reproduce**
1. Go to the MO dashboard
2. Click on the 'Risk Level' column header in the Case Table.
3. Notice that the arrow indicator changes on all column, not only the 'Risk Level' column

**Expected behavior**

**Screenshots**
Screenshots revealing sensitive information _must not_ be shared. If a screenshot is helpful in diagnosing and resolving the bug, please send this directly to a contact on the Recidiviz team.

**Desktop (please complete the following information):**
 - OS [e.g. Windows 10]: 
 - Browser [e.g. chrome, edge]: 
 - Version [e.g. 22]: 

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6]: 
 - OS: [e.g. iOS8.1]: 
 - Browser [e.g. stock browser, safari]: 
 - Version [e.g. 22]: 

**Additional context**

### PR DESCRIPTION
**Pull request by MeloncholyShiro**
## Description of the change

Fixed the arrow display when sorting, hopefully picked up the right solution. And you don't have multiple sorting choices

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #727 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/components/charts/new_revocations/CaseTable/CaseTable.js [hunk 1]
```diff
@@ -44,6 +44,7 @@ const CaseTable = () => {
   const [page, setPage] = useState(0);
   const { sortOrder, toggleOrder, comparator } = useSort();
   const { containerHeight, containerRef } = useContainerHeight();
+  const [toggledField, setToggledField] = useState();
```
Comment (Wmaileq) [inline src/components/charts/new_revocations/CaseTable/CaseTable.js:47]: We have this field already defined in `useSort` hook. Don't duplicate the logic.

**Review by Wmaileq — CHANGES_REQUESTED** [review]

File: src/components/charts/new_revocations/CaseTable/CaseTable.js (hunk 1 above)
Comment (MeloncholyShiro) [reply src/components/charts/new_revocations/CaseTable/CaseTable.js]: Fixed

File: src/components/charts/new_revocations/CaseTable/CaseTable.js [hunk 2]
```diff
@@ -64,6 +64,8 @@ const CaseTable = () => {
 
   const createSortableProps = (field) => ({
     order: sortOrder,
+    field,
```
Comment (Wmaileq) [inline src/components/charts/new_revocations/CaseTable/CaseTable.js:67]: Do not need to add extra dependencies. Your `Sortable` button should not know anything about other fields. (right now ALL the buttons will rerender on every `sort.field` change)
You can just move logic from line 51 of `src/components/charts/new_revocations/CaseTable/Sortable.js`
to the line below.

File: src/components/charts/new_revocations/CaseTable/Sortable.js [hunk 3]
```diff
@@ -39,7 +39,7 @@ SortableIcon.defaultProps = {
   order: null,
 };
 
-const Sortable = ({ children, order = null, onClick }) => (
+const Sortable = ({ children, order = null, field, sortField, onClick }) => (
```
Comment (Wmaileq) [inline src/components/charts/new_revocations/CaseTable/Sortable.js:42]: If you are working here, please take care of these inline styles on line 46 and move them to `.Sortable`.

Comment (MeloncholyShiro) [reply src/components/charts/new_revocations/CaseTable/Sortable.js]: Done

File: src/components/charts/new_revocations/CaseTable/CaseTable.js (hunk 2 above)
Comment (MeloncholyShiro) [reply src/components/charts/new_revocations/CaseTable/CaseTable.js]: Done

**Review by Wmaileq — APPROVED** [review]

File: src/components/charts/new_revocations/CaseTable/Sortable.css [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.Sortable {
```
Comment (jovergaag) [inline src/components/charts/new_revocations/CaseTable/Sortable.css]: In this repo we use `.scss` instead of `.css`.

**Review by jovergaag — APPROVED** [review]
I pushed up a commit renaming the Sortable.scss.

This looks great! Thank you @MeloncholyShiro

**Pull request merged**